server/charts/historian/values.yaml

```diff
@@ -4,7 +4,7 @@ imagePullPolicy: IfNotPresent
 
 historian:
   name: historian
-  image: prague.azurecr.io/historian:4049
+  image: fluidcr.azurecr.io/build/fluidframework/routerlicious/historian:latest
```

Comment:
Aren't you overriding these values in your deployment?

Reply:
Yup. I have overridden these values when doing the deployment and update these changes here. :)